packages/frontity-org-theme/src/components/styles/global-styles.tsx

```diff
@@ -17,44 +29,164 @@ const cssReset = css`
   h6,
   p,
   blockquote,
+  pre,
+  a,
+  abbr,
+  acronym,
   address,
   big,
   cite,
   code,
+  del,
+  dfn,
   em,
-  font,
   img,
+  ins,
+  kbd,
+  q,
+  s,
+  samp,
   small,
   strike,
+  strong,
   sub,
   sup,
-  li,
+  tt,
+  var,
+  b,
+  u,
+  i,
+  center,
+  dl,
+  dt,
+  dd,
   ol,
   ul,
+  li,
   fieldset,
   form,
   label,
   legend,
-  button,
   table,
   caption,
+  tbody,
+  tfoot,
+  thead,
   tr,
   th,
-  td {
-    border: none;
-    font-size: inherit;
-    line-height: inherit;
+  td,
+  article,
+  aside,
+  canvas,
+  details,
+  embed,
+  figure,
+  figcaption,
+  footer,
+  header,
+  hgroup,
+  menu,
+  nav,
+  output,
+  ruby,
+  section,
+  summary,
+  time,
+  mark,
+  audio,
+  video {
     margin: 0;
     padding: 0;
-    text-align: inherit;
+    border: 0;
+    font-size: 100%;
+    font: inherit;
+    vertical-align: baseline;
   }
-  blockquote::before,
-  blockquote::after {
+
+  footer,
+  header,
+  hgroup,
+  menu,
+  nav,
+  section {
+    display: block;
+  }
+
+  body {
+    line-height: 1;
+  }
+
+  ol,
+  ul {
+    list-style: none;
+  }
+
+  blockquote,
+  q {
+    quotes: none;
+  }
+
+  blockquote:before,
+  blockquote:after,
+  q:before,
+  q:after {
     content: "";
+    content: none;
   }
-  a,
-  path {
-    transition: all 0.15s linear;
+
+  table {
+    border-collapse: collapse;
+    border-spacing: 0;
+  }
+
+  img {
+    border-style: none;
+  }
+
+  button,
+  input,
+  optgroup,
+  select,
+  textarea {
+    font-family: inherit;
+    font-size: 100%;
+    line-height: 1.15;
+    margin: 0;
+  }
+
+  textarea {
+    overflow: auto;
+  }
+
+  button,
+  select {
+    text-transform: none;
+  }
+
+  button,
+  [type="button"],
+  [type="reset"],
+  [type="submit"] {
+    -webkit-appearance: button;
+  }
+
+  button::-moz-focus-inner,
+  [type="button"]::-moz-focus-inner,
+  [type="reset"]::-moz-focus-inner,
+  [type="submit"]::-moz-focus-inner {
+    border-style: none;
+    padding: 0;
+  }
+
+  button:-moz-focusring,
+  [type="button"]:-moz-focusring,
+  [type="reset"]:-moz-focusring,
+  [type="submit"]:-moz-focusring {
+    outline: 1px dotted ButtonText;
+  }
```

Comment:
what is that? why do we want a dotted outline on the button? 🙂

Reply:
This restores the focus outline styles unset by the previous rule.

Reply:
`ButtonText` is a `System Variable Color`. 

http://www.iangraham.org/books/xhtml1/appd/update-23feb2000.html